lib/hammer_cli_foreman_tasks/task.rb

```diff
@@ -44,14 +44,16 @@ class ListCommand < HammerCLIForeman::ListCommand
         field :result, _('Result')
         field :started_at, _('Started at'), Fields::Date
         field :ended_at, _('Ended at'), Fields::Date
+        field :duration, _('Duration')
         field :username, _('Owner')
 
         from :humanized do
           field :errors, _('Task errors'), Fields::List, :hide_blank => true
         end
       end
 
-      build_options
+
+      build_options 
```

Comment:
Let's not introduce more whitespace than we really need. There's a space after build_options and a blank line before it